assets/prefabs/characters/amphibious.prefab

```diff
@@ -0,0 +1,6 @@
+{
+  "parent": "Behaviors:testcharacter",
+  "MinionMove": {
+    "movementTypes": ["walking", "leaping", "swimming"]
+  }
```

Comment:
IIRC testcharacter already has a `MinionMove` component. With this we'll end up with the ambiguous state of having two and not knowing which one will be used.

Reply:
will mark this as resolved for now, we'll need to revisit this as a follow-up